docs/samples/custom/customized-urls/Readme.md

```diff
@@ -0,0 +1,107 @@
+# Customized URLs Sample
+
+In some situations, we want to deploy the inference service under `custom model`, and we just have the custom image but can not modify its server routes, this server may have one or more predict endpoints. This is a simple sample to show how to deploy this custom service.
+
+## A simple sample 
+
+* `app.py` is a simple flask server, it has tow endpoints: `customized_urls_test` and `hello_world`
+
+```yaml
+import os
+
+from flask import Flask
+
+app = Flask(__name__)
+@app.route('/customized/urls/demo/test/1')
+def customized_urls_test():
+    target = os.environ.get('TARGET', 'World')
+    return 'Hello {}. Func customized_urls_test is called!\n'.format(target)
+
+@app.route('/v1/models/customized-sample:predict')
+def hello_world():
+    target = os.environ.get('TARGET', 'World')
+    return 'Hello {}!\n'.format(target)
+
+if __name__ == "__main__":
+    app.run(debug=True,host='0.0.0.0',port=int(os.environ.get('PORT', 8080)))
+```
+
+* Build custom image.
+
+  Run the build command to build your image. Or you can use the the prebuild image:`iamlovingit/customized-urls:latest` 
+
+  ```shell
+  docker build -t ${your-dockerhub-id}/customized-urls:latest .
+  docker push ${your-dockerhub-id}/customized-urls:latest
+  ```
+
+* Config the Yaml
+
+  ```yaml
+apiVersion: serving.kubeflow.org/v1alpha2
+  kind: InferenceService
+  metadata:
+    labels:
+      controller-tools.k8s.io: "1.0"
+    name: customized-urls-sample
+  spec:
+    default:
+      predictor:
+        custom:
+          container:
+            name: hello
+            image: ${your-dockerhub-id}/customized-urls:latest
+            ports:
+            - containerPort: 5000
```

Comment:
knative by default expects to listen on port 8080 and your server is listening on that port, so this containerPort is not needed

Reply:
fixed